### PR TITLE
Add a e2e test for the singularity version --help command

### LINKED ISSUE #4102: [E2E] Improve coverage for version command
## Version of Singularity:

All.

### Expected behavior

All commands expected to succeed should be tested.
Part of issue #4078

### Actual behavior

- [ ] singularity version --help

### Steps to reproduce behavior

N/A.

### PR DESCRIPTION
**Pull request by gvallee**
**Description of the Pull Request (PR):**

This PR adds a simple test for the `singularity version --help` command. This is only meant to test the successful case in order to have better coverage for the e2e test and make sure there is no regression with the normal cases enabled by the CLI.

**This fixes or addresses the following GitHub issues:**

- Fixes #4102 

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: e2e/version/version.go [hunk 1]
```diff
@@ -96,6 +96,19 @@ func (c *ctx) testEqualVersion(t *testing.T) {
...
 
+// Test the help option
+func (c *ctx) testHelpOption(t *testing.T) {
+	c.env.RunSingularity(
+		t,
+		e2e.WithCommand("version"),
+		e2e.WithArgs("--help"),
+		e2e.ExpectExit(
+			0,
+			e2e.ExpectOutput(e2e.ContainMatch, "Show the version for Singularity"),
```
Comment (WestleyK) [inline e2e/version/version.go]: Does this only check the first line of the output?

**Review by WestleyK — APPROVED** [review]
Looks good to me :+1: Thanks!

File: e2e/version/version.go (hunk 1 above)
Comment (gvallee) [reply e2e/version/version.go]: Good question... I will investigate.

Comment (ikaneshiro) [reply e2e/version/version.go]: No, this will check the entire output for `"Show the version for Singularity"`

**Review by ikaneshiro — APPROVED** [review]

File: e2e/version/version.go (hunk 1 above)
Comment (gvallee) [reply e2e/version/version.go]: I will switch to `RegexMatch` after a suggestion from @ikaneshiro

**Review by WestleyK — APPROVED** [review]
Looks good to me! Thanks :+1:

**Pull request merged**